### PR TITLE
Refactored Counting Label duplicate code to avoid errors in future changes.

### PR DESCRIPTION
**Pull request by marcocapano**
This PR simple puts duplicate code into a block, resulting in the same effect but with no duplicates, meaning that future changes don't risk not to update that code everywhere.

### CODE REVIEW

File: Unwrap/Reusables/CountingLabel.swift [hunk 1]
```diff
@@ -47,24 +47,22 @@ class CountingLabel: UILabel {
...
         if targetValue > fromValue {
             // We need to add numbers to reach our target.
             currentValue += changeSpeed
-
-            if currentValue > targetValue {
-                // Target reached; clamp it and stop doing more work.
-                currentValue = targetValue
-                displayLink?.invalidate()
-            }
+            if currentValue > targetValue { stopWorking() }
```
Comment (twostraws) [inline Unwrap/Reusables/CountingLabel.swift]: This should be written across more lines: an empty line before the condition starts, then `stopWorking()` on a line of its own, and the closing brace on a line of its own. Unless I missed something, the only time Unwrap code puts conditions on one line only when using `return` as part of a `guard` early return.

File: Unwrap/Reusables/CountingLabel.swift [hunk 2]
```diff
@@ -47,24 +47,22 @@ class CountingLabel: UILabel {
...
         } else {
             // We need to subtract numbers to reach our target.
             currentValue -= changeSpeed
-
-            if currentValue < targetValue {
-                // Target reached; clamp it and stop doing more work.
-                currentValue = targetValue
-                displayLink?.invalidate()
-            }
+            if currentValue < targetValue { stopWorking() }
```
Comment (twostraws) [inline Unwrap/Reusables/CountingLabel.swift]: Same as above.

File: Unwrap/Reusables/CountingLabel.swift [hunk 3]
```diff
@@ -47,24 +47,22 @@ class CountingLabel: UILabel {
 
     /// Called every time a frame draw happens thanks to CADisplayLink, so its our chance to adjust our current value towards our target value, then update our label.
     @objc func updateCount() {
+        
+        // Target reached; clamp it and stop doing more work.
+        let stopWorking = {
```
[the remark this answers is not in the document]
Comment (twostraws) [reply Unwrap/Reusables/CountingLabel.swift]: Yes, a function would be perfect – thank you!

File: Unwrap/Reusables/CountingLabel.swift (hunk 1 above)
Comment (twostraws) [reply Unwrap/Reusables/CountingLabel.swift]: Awesome. If you could make the closure into a function and fix the new lines, then push up your changes, I can merge.

**Comment by marcocapano** [comment]
Ok @twostraws I made those changes, let me know if it looks good to you. And sorry if it took a while to do it👍🏻

**Comment by twostraws** [comment]
Thank you!

**Pull request merged**